cmd/tailscale/cli/set.go

```diff
@@ -233,6 +236,16 @@ func runSet(ctx context.Context, args []string) (retErr error) {
 			}
 		}
 	}
+
+	if setArgs.relayServerPort != "" {
+		uport, err := strconv.ParseUint(setArgs.relayServerPort, 10, 16)
+		if err != nil {
+			return fmt.Errorf("failed to set relay server port: %v", err)
+		}
+		port := int(uport)
+		maskedPrefs.Prefs.RelayServerPort = &(port)
```

Comment:
drop parens around port. I didn't even know that was permitted :)

but it looks like it does something it doesn't do (make a new pointer, not just `&port`)

alternatively, `maskedPrefs.Prefs.RelayServerPort = ptr.To(int(uport))` if you want

Reply:
Parens were unintentional, updated in https://github.com/tailscale/tailscale/pull/15594/commits/1592c6beabcaf5c5d8e6e14d7d54579f3cca8c67.